components/ContactForm/index.tsx

```diff
@@ -55,9 +55,8 @@ export const ContactForm = () => {
     setEmailErr(emailValid);
 
     const errArray = [nameValid, messageValid, emailValid];
-    let isValid = true;
-    errArray.forEach((error) => {
-      if (error !== null) isValid = false;
+    const isValid = errArray.every((error) => {
+      error === null;
```

Comment:
This will always return `false`. You forgot the `return` statement. In arrow functions, though, you can use a shortcut. If you omit the function body `{...}` and directly write the expression - it will be automatically returned:

```ts
error => error === null
```
Which is equivalent to

```ts
error => {
  return error === null
} 
```